Podfile.lock

```diff
@@ -11,6 +11,6 @@ SPEC REPOS:
 SPEC CHECKSUMS:
   SwiftLint: 22ccbbe3b8008684be5955693bab135e0ed6a447
 
-PODFILE CHECKSUM: f12283feb35e1e67ec7946a36c30e748cd6ac057
+PODFILE CHECKSUM: 9e9fa74647c6b2fb89bbf8f6a7f0102a43ae068b
 
-COCOAPODS: 1.9.1
+COCOAPODS: 1.8.4
```

Comment:
would it be possible to update your CocoaPods. version to `1.9.1` @eddyteddy3  - otherwise we'll constantly be getting this mismatch

Reply:
i have updated my CocoaPods Version